glusterd2/commands/snapshot/snapshot-delete.go

```diff
@@ -91,14 +91,6 @@ func snapshotDeleteStore(c transaction.TxnCtx) error {
 			Warn("failed to delete volfiles of snapshot")
 	}
 
-	//Snapshot can be deleted even if it is activated, hence regenerating volfiles
```

Comment:
Snapshot can be deleted even when a volume is activated (To support auto-delete, delete all etc), so volfiles has to be deleted as well if snapshot is activated during a delete.
Earlier patch set was deleting brick volfiles during a delete operation for volume, hence I didn't explicitly commented it.

Reply:
ack